Jenkinsfile

```diff
@@ -11,14 +11,33 @@ pipeline {
         dockerfile true
     }
 
+    parameters {
+        string(name: "REMOTE_SWF_LINK", defaultValue: "https://github.com/Pelagicore/software-factory.git")
+        string(name: "REMOTE_SWF_BP_LINK", defaultValue: "https://github.com/Pelagicore/software-factory-blueprint.git")
```

Comment:
I'm a bit skeptical to this particular parameter. Since this Jenkinsfile is in the BP repo already, it should not be needed. We should be able to do "checkout scm" or similar and then work with that. Perhaps we can check out the code in a subdirectory and copy it later to the SWF build or something?